open-wc-testing-config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	"extends": "./lit-config.js",
+	"env": {
+	  "mocha": true
+	}
+  };
+  
```

Comment:
I removed the disabling of `no-invalid-this` and `prefer-arrow-callback`... not sure if they were added for a good reason that still applies.

Reply:
Here's the `wct-lit-config.js` for reference:

```
module.exports = {
  "extends": "./lit-config.js",
  "env": {
    "mocha": true,
    "node": true
  },
  "globals": {
    "assert": true,
    "fixture": true,
    "flush": true,
    "expect": true,
    "MockInteractions": true,
    "sinon": true,
    "WCT": true
  },
  "rules": {
    "no-invalid-this": 0,
    "prefer-arrow-callback": 0
  }
};
```

Reply:
I'm not 100% sure, but I think there were certain specific scenarios where arrow callbacks were problematic with mocha.  I eventually found this which confirmed there can be problems.  

https://mochajs.org/#arrow-functions

Reply:
Cool... well since the open-wc testing stuff uses arrow functions liberally, I'm going to assume we're good. It doesn't rely on globals for assert/expect/etc. but you instead import them, so maybe that solves things.

Reply:
Sounds good.  It might have been the `this.test.fullTitle()` API call that our visual-diff tests rely on that seemingly would have blown up trying to use arrow functions.

Reply:
Right. It's easy enough to add it if we end up needing it.

Reply:
Yup so just running into this with testing my `visual-diff` changes in the template, both in the visual-diff .js file. Without `no-invalid-this`, there is a failure at `this.test.fullTitle()`, and without `node` there are `no-undef` lint failures (`'require' is not defined` and `'__dirname' is not defined`)

Reply:
Boo. Ok I'll add them back in. Seems like ultimately though we may want a different config file for the mocha/Karma tests vs. the visual-diff ones. And/or find a way to convert the visual diff ones to behave more like the Karma ones.